inference-engine/tests/ngraph_functions/src/gru_cell.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include <memory>
+
+#include "ngraph_functions/builders.hpp"
+
+namespace ngraph {
+namespace builder {
+
+std::shared_ptr<ngraph::Node> makeGRUCell(const OutputVector& in,
+                                          const std::vector<ngraph::Shape>& WRB,
+                                          std::size_t hidden_size,
+                                          const std::vector<std::string>& activations,
+                                          const std::vector<float>& activations_alpha,
+                                          const std::vector<float>& activations_beta,
+                                          float clip,
+                                          bool linear_before_reset) {
+    std::vector<float> empty;
+    auto W = ngraph::builder::makeConstant(in[0].get_element_type(), WRB[0], empty, true);
```

Comment:
@ilyachur , did we plan to get rid of ngraph builders?

Reply:
it's our ngraph builders in the tests, requested by @iefode